packages/@ourworldindata/grapher/src/core/Grapher.tsx

```diff
@@ -1993,18 +2008,18 @@ export class Grapher
                 title: this.isPlaying ? `Pause` : `Play`,
                 category: "Timeline",
             },
-            {
-                combo: "f",
-                fn: (): void => this.toggleFacetControlVisibility(),
-                title: `Toggle Faceting`,
-                category: "Chart",
-            },
             {
                 combo: "l",
                 fn: (): void => this.toggleYScaleTypeCommand(),
                 title: "Toggle Y log/linear",
                 category: "Chart",
             },
+            {
+                combo: "w",
+                fn: (): void => this.toggleFullScreenMode(),
+                title: `Toggle full-screen mode`,
+                category: "Chart",
+            },
```

Comment:
Note for reviewers: A little treat for our power users hehe. Not sure how much I like the `w` (for 'window'). Suggestions welcome!